### PR TITLE
Implement popup window

### PR DESCRIPTION
**Pull request by Zeph180**
- Create a popup window
- Populate a popup window with items from API
- Implement a close button.
- List comments fetched from popup api

### CODE REVIEW

File: src/comment-modal.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+          const commentP = document.createElement('p');
+          commentP.textContent = `${comment.creation_date}: ${comment.username} ${comment.comment}`;
+          commentsSection.appendChild(commentP);
+        });
+      });
+  }
+
+  updateComments();
+  const commentInterval = setInterval(updateComments, 10000);
+
```
Comment (djo1975) [inline src/comment-modal.js:53]: This very smart way to refresh comments

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(217 202 202);
+}  */
```
Comment (djo1975) [inline src/style.css:151]: Please remove this unused code

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+/*
+.comment-modal h4 {
+  position: relative;
+  font-size: xx-large;
+} */
+
+/* .add-comment {
+  margin-top: 1rem;
+  margin-bottom: 5rem;
+} */
```
Comment (djo1975) [inline src/style.css:232]: Please remove this unused code

**Review by djo1975 — APPROVED** [review]
Hello @Zeph180 
Your code looks good. There is nothing else to say then merge it.

File: src/comment-modal.js (hunk 1 above)
Comment (Zeph180) [reply src/comment-modal.js:53]: @djo1975 
I am humbled

Comment (Zeph180) [reply src/comment-modal.js:53]: @djo1975 
I am humbled

File: src/style.css (hunk 2 above)
Comment (Zeph180) [reply src/style.css:151]: Thanks @djo1975

Comment (Zeph180) [reply src/style.css:151]: Thanks @djo1975

Comment (Zeph180) [reply src/style.css:151]: Thanks @djo1975

**Pull request merged**